builtin/credential/okta/path_config.go

```diff
@@ -278,9 +317,20 @@ func (c *ConfigEntry) OktaClient() *okta.Client {
 		baseURL = c.BaseURL
 	}
 
-	// We validate config on input and errors are only returned when parsing URLs
-	client, _ := okta.NewClientWithDomain(cleanhttp.DefaultClient(), c.Org, baseURL, c.Token)
-	return client
+	if c.Token != "" {
+		client, err := oktanew.NewClient(context.Background(),
```

Comment:
I was going to ask about context handling improvements, but it looks like the Okta SDK doesn't even do anything with the context, so nvm!  https://github.com/okta/okta-sdk-golang/blob/7334d45e99bc09e0dbe209ef961aa40827e9d012/okta/okta.go#L55